ruff_cli/pyproject.toml

```diff
@@ -13,7 +13,7 @@ maintainers = [
   { name = "Charlie Marsh", email = "charlie.r.marsh@gmail.com" },
 ]
 requires-python = ">=3.7"
-license = { file = "LICENSE" }
+license = { file = "../LICENSE" }
```

Comment:
The `tool.setuptools` references below would need to be changed as well (though I think they're currently being ignored -- please change them though).